examples/site/howto_local_development/README.md

```diff
@@ -34,25 +34,86 @@ would use:
 
 ```shell
 cd $HOME
-git clone https://github.com/microsoft/vcpkg
-(cd vcpkg && ./bootstrap.sh)
+git clone -q https://github.com/microsoft/vcpkg
+# Expected output: none
+(cd vcpkg && ./bootstrap-vcpkg.sh --disableMetrics --useSystemBinaries)
```

Comment:
Your call, but I believe the `cd` isn't needed: `./vcpkg/bootstrap-vcpkg.sh ...` should work.

Reply:
Fixed.